app/views/govuk_publishing_components/component_guide/index.html.erb

```diff
@@ -105,5 +105,5 @@
 </ul>
 
 <div class="component-markdown">
-  <p class="govuk-body">If you cannot find a suitable component consider extending an existing component or <a href="https://github.com/alphagov/govuk_publishing_components/blob/master/docs/develop-component.md">creating a new one</a>.</p>
+  <p class="govuk-body">If you cannot find a suitable component consider extending an existing component or <a href="/alphagov/govuk_publishing_components/blob/master/docs/generate-a-new-component.md">creating a new one</a>.</p>
```

Comment:
Nice catch 👍

Reply:
Yes, but the link doesn't work, it still needs `https://github.com/` at the start.

Reply:
As Andy mentioned, the link is still broken.